packages/shared/src/components/post/PostContent.tsx

```diff
@@ -253,119 +270,170 @@ export function PostContent({
+  );
+  const postFeedFiltersOnboarding = showMyFeedArticleAnonymous && (
+    <PostFeedFiltersOnboarding
+      hasNavigation={hasNavigation}
+      version={myFeedArticleAnonymousVersion}
+      postPreviousNext={postPreviousNext}
+      onInitializeOnboarding={onInitializeOnboardingClick}
+    />
+  );
+
   return (
     <Wrapper
-      className={className}
+      className={classNames(
+        className,
+        !isPostFeedFiltersOnboardingV1 && ' tablet:flex-row',
+      )}
       aria-live={subject === ToastSubject.PostContent ? 'polite' : 'off'}
     >
-      <PostContainer
-        className={classNames(
-          'relative',
-          isFixed && 'pt-16',
-          isFixed && !hasNavigation && 'tablet:pt-0',
+      {isPostFeedFiltersOnboardingV1 && postFeedFiltersOnboarding}
+      <ConditionalWrapper
+        condition={isPostFeedFiltersOnboardingV1}
+        wrapper={(children) => (
+          <section className="flex flex-col tablet:flex-row flex-1">
+            {children}
+          </section>
         )}
       >
-        <PostNavigation
-          onPreviousPost={onPreviousPost}
-          onNextPost={onNextPost}
-          className={classNames(
-            !hasNavigation && !isFixed && 'tablet:pt-0',
-            !hasNavigation ? 'flex laptop:hidden' : 'flex',
-            padding,
-            position,
-            isFixed &&
-              'z-3 w-full bg-theme-bg-secondary border-b border-theme-divider-tertiary px-6 top-0 -ml-8',
-            isFixed && styles.fixedPostsNavigation,
-          )}
-          shouldDisplayTitle={isFixed}
-          post={postById.post}
-          onReadArticle={onReadArticle}
-          onClose={onClose}
-          isModal={isModal}
-          onBookmark={toggleBookmark}
-          onShare={onShare}
-        />
-        <h1
-          className="my-6 font-bold break-words typo-large-title"
-          data-testid="post-modal-title"
-        >
-          {postById.post.title}
-        </h1>
-        {postById.post.summary && (
-          <PostSummary summary={postById.post.summary} />
-        )}
-        <TagLinks tags={postById.post.tags || []} />
-        <PostMetadata
-          createdAt={postById.post.createdAt}
-          readTime={postById.post.readTime}
-          className="mt-4 mb-8"
-          typoClassName="typo-callout"
-        />
-        <a
-          {...postLinkProps}
-          className={classNames(
-            'block overflow-hidden mb-10 rounded-2xl cursor-pointer',
-            styles.clickableImg,
-          )}
-          style={{ maxWidth: '25.625rem' }}
-        >
-          <LazyImage
-            imgSrc={postById.post.image}
-            imgAlt="Post cover image"
-            ratio="49%"
-            eager
-            fallbackSrc="https://res.cloudinary.com/daily-now/image/upload/f_auto/v1/placeholders/1"
-          />
-        </a>
-        {postById.post?.toc?.length > 0 && (
-          <PostToc
+        <>
```

Comment:
I think we can remove the fragment here.

Reply:
Ah technically yes, but the conditionalWrapper expects valid ReactElements which can only be one parent.
This one includes the PostWidgets as a separate element, thus being wrapped in a fragment.

Hope that makes sense?

Reply:
Is it not possible to make the `children` a `ReactNode` as we commonly use?

If not, then, in that case, this is really necessary. Thanks for letting me know! 🚢

Reply:
This will give the error that ReactNode is not a valid JSX Element.
Think this is quite what we would expect when passing multi-high level conditions right.

Let's leave this for now, we can always modify later?

Reply:
I agree. Let's keep this as the current implementation 🚢